### PR TITLE
Singleton utils

### PR DESCRIPTION
**Pull request by akhil-ghatiki**
# Description
Implemented Bill Pugh Singleton implementation for AddTextEnhancementOptionUtils, BrushUtils, ColorUtils, CommonCodeUtils. 
Modified the corresponding usages of these methods.
Validate the PR per commit as count of files changed is high.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
ran ./gradlew test for each commit

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/activity/ImageEditor.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ void initValues() {
         mFilterUris = getIntent().getStringArrayListExtra(IMAGE_EDITOR_KEY);
         mDisplaySize = mFilterUris.size();
         mFilterItems = getFiltersList(this);
-        mBrushItems = getBrushItems();
+        mBrushItems = getInstance().getBrushItems();
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/activity/ImageEditor.java]: use `BrushUtils.getInstance` here. It'll be easier to read.

File: app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java [hunk 2]
```diff
@@ -288,16 +286,16 @@ private void setTextAndActivateButtons(String path) {
 
     @Override
     public void onPopulate(ArrayList<String> paths) {
-        populateUtil(mActivity, paths, this, mLayout, mLottieProgress, mRecyclerViewFiles);
+        getInstance().populateUtil(mActivity, paths, this, mLayout, mLottieProgress, mRecyclerViewFiles);
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java]: hre also, use `CommonCodeUtils.getInstance`

File: app/src/main/java/swati4star/createpdf/activity/ImageEditor.java (hunk 1 above)
Comment (akhil-ghatiki) [reply app/src/main/java/swati4star/createpdf/activity/ImageEditor.java]: @Swati4star made the changes

File: app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java (hunk 2 above)
Comment (akhil-ghatiki) [reply app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java]: @Swati4star made the changes

**Pull request merged**